locksmith/__tests__/token_middleware.test.js

```diff
@@ -0,0 +1,163 @@
+var httpMocks = require('node-mocks-http')
+var tokenMiddleware = require('../src/token_middleware')
+
+let request, response, validAuthorizationHeader
+
+beforeAll(() => {
+  request = httpMocks.createRequest()
+  response = httpMocks.createResponse()
+  validAuthorizationHeader = {
+    'Authorization':
+      'Bearer eyJhbGciOiJub25lIiwidHlwIjoiSldUIn0.eyJsb2NrIjp7ImFkZHJlc3MiOiJqcWE2ZG5wMSIsIm5hbWUiOiJvaW9pb2kiLCJleHBpcmF0aW9uRHVyYXRpb24iOjI1OTIwMDAsImtleVByaWNlIjoiMTAwMDAwMDAwMDAwMDAwMDAiLCJtYXhOdW1iZXJPZktleXMiOjEwLCJvd25lciI6IjB4QWFBZEVFRDRjMEI4NjFjQjM2ZjRjRTAwNmE5QzkwQkEyRTQzZmRjMiJ9LCJpYXQiOjE1NDYxMzA4MzUsImV4cCI6MTU0NjEzMDgzOCwiaXNzIjoiMHhBYUFkRUVENGMwQjg2MWNCMzZmNGNFMDA2YTlDOTBCQTJFNDNmZGMyIn0.0xd36a7842eda1b2267c08b204407f9e71024a2152ae772882a1fa41a02696aee578688f02ef6c301451de95793bde669198359f0820fb35f6ba1303e89727a52601',
```

Comment:
Maybe extract this mock jwt and lock addresses into a file in tests/ and import it into the tests? Since the same value is used in multiple tests